docs/12-upgrading/01-upgrade-to-two.md

```diff
@@ -18,14 +51,32 @@ session.db.find(...);
 
 ### Authenticated user information retrieval
 
-In Serverpod 2.0, we have removed the getters `scopes` and `authenticatedUser` from session. This information is now retrievable through the `authenticationInfo` getter as fields of the returned object. 
+In Serverpod 2.0, we have removed the getters `scopes` and `authenticatedUser` from session. This information is now retrievable through the `authenticated` getter as fields of the returned object.
+
+```dart
+final auth = await session.authenticated;
+
+//Read authenticated userId
+final userId = auth?.userId;
+
+//Read scopes
+final scopes = auth?.scopes;
+```
```

Comment:
Suggestion: add a "before change" example as well so that it is easy to understand if you are affected.